src/lib/components/artwork_grids/infinite_scroll_grid.tsx

```diff
@@ -35,52 +28,56 @@ export const PageEndThreshold = 1000
  *   - the calculation currently only takes into account the size of the image, not if e.g. the sale message is present
  */
 
-type Artworks = Array<{
-  __id: string,
-  image: {
-    aspect_ratio: number | null,
-  } | null,
-}>
+type Artworks = Array<
+  {
+    __id: string,
+    image:
+      | {
+        aspect_ratio: number | null,
+      }
+      | null,
+  }
+>
 
 interface Props extends ArtistRelayProps, GeneRelayProps {
   /** The direction for the grid, currently only 'column' is supported . */
-  sectionDirection: string;
```

Comment:
Looks like I was wrong - we don't want commas.

My issues came up when I was using the typings parser inside Monaco for this page: http://danger.systems/js/editor.html

Basically the [Danger DTS](https://github.com/danger/danger-js/blob/master/source/danger.d.ts) would fail with no error messaging about why it failed. Eventually I found [dtslint](https://github.com/Microsoft/dtslint) which I started applying to Danger's interfaces and found that it provided errors on commas as an issue. Now it's entirely free of them.